src/subapps/dataExplorer/ProjectSelector.tsx

```diff
@@ -0,0 +1,86 @@
+import { ProjectResponseCommon } from '@bbp/nexus-sdk';
+import { useNexusContext } from '@bbp/react-nexus';
+import { AutoComplete, Input, notification } from 'antd';
+import React, { useState } from 'react';
+import { useQuery } from 'react-query';
+import './styles.less';
+
+interface Props {
+  onSelect: (orgLabel?: string, projectLabel?: string) => void;
+}
+
+export const ProjectSelector: React.FC<Props> = ({ onSelect }: Props) => {
+  const { data: projects } = useProjects();
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const allOptions = [
+    { value: AllProjects, label: AllProjects },
+    ...(projects?.map(projectToOption) ?? []),
+  ].filter(project => project.value.includes(searchTerm));
+
+  return (
+    <div className="search-container">
+      <span className="label">Show me </span>
+      <AutoComplete
+        options={allOptions}
+        style={{ width: 200 }}
+        defaultValue={AllProjects}
+        onSearch={text => {
+          setSearchTerm(normalizeSearchTerm(text));
+        }}
+        onSelect={text => {
+          if (text === AllProjects) {
+            onSelect(undefined, undefined);
+          } else {
+            const [org, project] = text.split('/');
+            onSelect(org, project);
+          }
+        }}
+        aria-label="project-filter"
+        bordered={false}
+        className="search-input"
+        popupClassName="search-menu"
+      >
+        <Input.Search size="middle" />
+      </AutoComplete>
+    </div>
+  );
+};
+
+export const AllProjects = 'All projects';
```

Comment:
can we rename it as constant (uppercase) so we can understand by the format (and it will be consistent as in different places in the APP)